Adds routing bug doc text to the 4.9 RNs (#36994)
openshift/openshift-docs: release_notes/ocp-4-9-release-notes.adoc

*Routing*

> * Previously, the config drift seemed to happen when the Cluster Network Operator (CNO) attempted to sanitize the proxy configuration, specifically the `no_proxy` config. Consequently, this resulted in a specific IPv6 CIDR missing from the `no_proxy`. This fix implements logic that updates the dual stack (IPV4 and IPV6) for all scenarios. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1981975[*BZ#1981975*])

Reviewer: ```suggestion
* Previously, there was config drift when the Cluster Network Operator (CNO) attempted to sanitize the proxy configuration, specifically the `no_proxy` config. This resulted in a specific IPv6 CIDR missing from the `no_proxy`. This fix implements logic that updates the dual stack (IPV4 and IPV6) for all scenarios. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1981975[*BZ#1981975*])
```

> * The HAProxy router should expose prometheus metrics for a route by enabling `openshift-monitoring` to pull the metrics, but it was failing intermittently. This caused instability when CI conformance tests were performed. This fix adds the simple selector for active data only in the URL, which reduces the data response from the prometheus server. Now, continuous execution in the CI process are more stable. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1969471[*BZ#1969471*])

Reviewer: I *think* this bug is just for internal testing. Can you ask the person who wrote it up to see if it really needs to be included?

Reviewer: Nevermind, I see she was questioning this too. I think leave it out.
